sources/packages/backend/apps/api/src/route-controllers/student-scholastic-standings/student-scholastic-standings.controller.service.ts

```diff
@@ -57,6 +60,29 @@ export class ScholasticStandingControllerService {
     };
   }
 
+  /**
+   * Get Scholastic Standing summary details.
+   * @param studentId student id to retrieve the scholastic standing summary.
+   * @returns Scholastic Standing summary details.
+   */
+  async getScholasticStandingSummary(options?: {
+    studentId: number;
+  }): Promise<ScholasticStandingSummaryDetailsAPIOutDTO> {
+    const scholasticStandingSummary =
+      await this.studentScholasticStandingsService.getScholasticStandingSummary(
+        options?.studentId,
+      );
+    const SFASUnsuccessfulCompletionWeeks =
+      await this.sfasIndividualService.getSFASTotalUnsuccessfulCompletionWeeks(
+        options?.studentId,
+      );
+    return {
+      lifetimeUnsuccessfulCompletionWeeks:
+        +(scholasticStandingSummary?.totalUnsuccessfulWeeks ?? 0) +
```

Comment:
I would like to suggest having student scholastic standing unsuccessful weeks and sfas scholastic standing unsuccessful weeks as constant for better readability.